app/services/preserved_object_handler_results.rb

```diff
@@ -107,25 +103,29 @@ def report_results
         candidate_workflow_results << r
       end
     end
-    stack_trace = caller(1..1).first[/`\S+/].chop[1..-1]
+    stack_trace = caller(0..1).join("\n")
```

Comment:
my range stuff is a bit muzzy, and my caller knowledge more so  -- is this basically taking the first 2 lines from the caller stack?

Reply:
yup first 2 lines

Reply:
Yep, `caller` returns an Array of String lines, and this grabs the first two elements.